pkg/client/cli/cmd_config.go

```diff
@@ -0,0 +1,73 @@
+package cli
+
+import (
+	"encoding/json"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+	empty "google.golang.org/protobuf/types/known/emptypb"
+
+	"github.com/telepresenceio/telepresence/v2/pkg/client"
+	"github.com/telepresenceio/telepresence/v2/pkg/client/cli/ann"
+	"github.com/telepresenceio/telepresence/v2/pkg/client/cli/output"
+	"github.com/telepresenceio/telepresence/v2/pkg/client/cli/util"
+	"github.com/telepresenceio/telepresence/v2/pkg/client/userd"
+	"github.com/telepresenceio/telepresence/v2/pkg/filelocation"
+)
+
+func configCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use: "config",
+	}
+	cmd.AddCommand(configViewCommand())
+	return cmd
+}
+
+func configViewCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:               "view",
+		Args:              cobra.NoArgs,
+		PersistentPreRunE: output.DefaultYAML,
+		Short:             "View current Telepresence configuration",
+		RunE:              configView,
+	}
+	cmd.Flags().BoolP("client-only", "c", false, "Only view config from client file.")
+	return cmd
+}
+
+func configView(cmd *cobra.Command, _ []string) error {
+	var cfg userd.SessionConfig
+	clientOnly, _ := cmd.Flags().GetBool("client-only")
+	if clientOnly {
+		// Unable to establish a session, so try to convey the local config instead. It
+		// may be helpful in diagnosing the problem.
+		ctx := cmd.Context()
+		cfgDir, err := filelocation.AppUserConfigDir(ctx)
+		if err != nil {
+			return err
+		}
+		cfg.Config = client.GetConfig(cmd.Context())
+		cfg.ClientFile = filepath.Join(cfgDir, client.ConfigFile)
+		// TODO: Also retrieve kubeconfig extension
```

Comment:
we might want to actually do this before merging, seems incomplete to not include the kubeconfig extension.

Reply:
Yep, I completely forgot about that one.